screen-orientation/resources/sandboxed-iframe-locking.html

```diff
@@ -20,7 +20,12 @@
   }
 
   screen.orientation.unlock();
-  await document.exitFullscreen();
+  try {
+    await document.exitFullscreen();
+  } catch (error) {
+    data.result = "errored";
+    data.name = "fullscreen exit failed";
```

Comment:
Although FS API generally only rejects with `TypeError`, I wonder if it might be safe to post back `error.name`?